### PR TITLE
update ui components for mobile

### LINKED ISSUE #1250: V4: Update UI components for Mobile
Some UX tweaks/polishing needed over time, like buttons running out of the screen or some default-zoom action upon tapping “search"

https://drive.google.com/file/d/1q0lxISUHisUuYejK4MvVA0GXIrnIbWnR/view?usp=sharing

The V4 market works pretty well on mobile, but there are some UI glitches we need to fix.

### PR DESCRIPTION
**Pull request by Abrom8**
Fixes #1250

Changes proposed in this PR:
Kept changes to a minimum.

- fix mobile view of filters (global search + profile assets view)
- fix  mobile view of tabs on profile page
- fix to prevent zoom on input focus for the search bar

### CODE REVIEW

File: src/components/Header/SearchBar.module.css [hunk 1]
```diff
@@ -26,6 +26,9 @@
 }
 
 .input {
+  font-size: var(
+    --font-size-large
```
Comment (kremalicious) [inline src/components/Header/SearchBar.module.css:30]: good point, and we should revisit this for all text form fields actually. But in end we just need to get to computed font-size of 16px, while this here leads to a size of around 18px which appears way too big up there. Sao to move forward quickly with this PR, and as there is a code comment, we can get away with hardcoding to `font-size` to `16px` here

Comment (Abrom8) [reply src/components/Header/SearchBar.module.css]: `font-size` is now hardcoded to `16px`.

**Comment by kremalicious** [comment]
sorry, old PR, think you have to merge or rebase against `main`, as some subgraph structure had changed and that might be the issue leading Vercel to failed deployments

**Comment by Abrom8** [comment]
@kremalicious looks like Vercel deployment works now

**Review by kremalicious — APPROVED** [review]
cool, nice surgical fixes

**Pull request merged**